adal/src/main/java/com/microsoft/aad/adal/UserInfo.java

```diff
@@ -122,9 +122,9 @@ static UserInfo getUserInfoFromBrokerResult(final Bundle bundle) {
     }
 
     /**
-     * Gets unique userid.
+     * Gets unique user id.
      * 
-     * @return
+     * @return String
```

Comment:
This is a public interface, we should document more. i.e. The unique id representing an user.